### PR TITLE
Rewritten and improved edgectl install console messaging

### PR DESCRIPTION
**Pull request by brucehorn**
Addresses issues in the console flows for edgectl install as noted in https://docs.google.com/document/d/1ZVEMpXzaAUIKZ3hts7UMpaYYbUMTic1J1qAR1l5u1xU/edit#heading=h.aj4tj6tj52pk

Several screenshots shown inline; the full list attached as a zip file.

[Screenshots1.4.1.zip](https://github.com/datawire/ambassador/files/4461931/Screenshots1.4.1.zip)

**Successful install:**

Successful install, new stack

**Incompatible CRDs:**

IncompatibleCRDVersionsError

### CODE REVIEW

File: cmd/edgectl/aes_login.go [hunk 1]
```diff
@@ -3,12 +3,12 @@ package main
...
 	"crypto/rsa"
 	"fmt"
+	"github.com/pkg/browser"
 	"time"
 
 	"github.com/datawire/ambassador/pkg/k8s"
 	"github.com/dgrijalva/jwt-go"
 	"github.com/gookit/color"
-	"github.com/pkg/browser"
 	"github.com/pkg/errors"
```
Comment (inercia) [inline cmd/edgectl/aes_login.go:12]:
```suggestion
	"github.com/pkg/errors"
```

File: cmd/edgectl/aes_login.go [hunk 2]
```diff
@@ -3,12 +3,12 @@ package main
 import (
 	"crypto/rsa"
 	"fmt"
+	"github.com/pkg/browser"
```
Comment (inercia) [inline cmd/edgectl/aes_login.go:6]:
```suggestion
```

**Comment by ark3** [comment]
It looks like you removed the `go.mod` changes. Good! You should also remove the `go.sum` changes.

**Comment by brucehorn** [comment]
A couple of new screenshots with cleaned-up flow:

**Successful:**

Successful installation

**Existing:**

Existing installation

**Pull request merged**